src/PointwiseFunctions/AnalyticSolutions/NewtonianEuler/IsentropicVortex.hpp

```diff
@@ -127,8 +130,8 @@ class IsentropicVortex : public MarkAsAnalyticSolution {
     static type lower_bound() noexcept { return 0.0; }
   };
 
-  using options = tmpl::list<AdiabaticIndex, Center, MeanVelocity,
-                             PerturbAmplitude, Strength>;
+  using options = tmpl::list<AdiabaticIndex, Center, MeanVelocity, Strength,
+                             PerturbAmplitude>;
```

Comment:
Remove PerturbAmplitude in 2d.

Reply:
Would using a `tmpl::conditional_t` suffice?

Reply:
Yes.